lib/ood_core/job/adapters/slurm.rb

```diff
@@ -882,6 +907,20 @@ def parse_job_info(v)
               end
             end
 
+            if v[:min_memory] && !v[:min_memory].empty?
+              # Slurm uses per CPU memory if --mem-per-cpu with 'Mc' output
+              # or uses per node if --mem with 'M' output
+              if v[:min_memory].end_with?('c')
+                # memory per CPU
+                memory_per = :cpu
```

Comment:
Where did you get this from? I'm unable to replicate this or see it in any job at OSC.

Reply:
https://slurm.schedmd.com/sbatch.html

I was trying to work off the slurm docs, i can yank it though.

Reply:
I know you can request `--mem-per-cpu` but I don't see in the `squeue` output where it's labeled as `Mc`, even when I try to replicate submitting jobs with `--mem-per-cpu` set, it always reports memory as `XXXXM`.

Were you able to actually see `Mc` in the `squeue` output?